### PR TITLE
EuroLinux 9 chroot config

### PR DESCRIPTION
**Pull request by istiak101**
EuroLinux 9 mock chroot config.

Tagging @AlexBaranowski for review.

### CODE REVIEW

File: mock-core-configs/etc/mock/templates/eurolinux-9.tpl [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+[ha-all]
+name=EuroLinux 9 HighAvailability All
+baseurl=https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/$basearch/certify-HighAvailability/all/
+gpgcheck=1
+enabled=1
+gpgkey=file:///usr/share/distribution-gpg-keys/eurolinux/RPM-GPG-KEY-eurolinux9
+
+[rs-all]
+name=EuroLinux 9 PowerTools All
+baseurl=https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/$basearch/certify-ResilientStorage/all/
```
Comment (AlexBaranowski) [inline mock-core-configs/etc/mock/templates/eurolinux-9.tpl:62]: Please consider the following change:

```
baseurl=https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/$basearch/ResilientStorage/all/
```

File: mock-core-configs/etc/mock/templates/eurolinux-9.tpl [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+baseurl=https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/$basearch/certify-PowerTools/all/
+gpgcheck=1
+enabled=1
+gpgkey=file:///usr/share/distribution-gpg-keys/eurolinux/RPM-GPG-KEY-eurolinux9
+
+# There is no HA and RS for i686 or aarch64
+{% if target_arch == 'x86_64' %}
+[ha-all]
+name=EuroLinux 9 HighAvailability All
+baseurl=https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/$basearch/certify-HighAvailability/all/
```
Comment (AlexBaranowski) [inline mock-core-configs/etc/mock/templates/eurolinux-9.tpl:55]: Please consider the following change:

```
baseurl=https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/$basearch/HighAvailability/all/
```

**Review by AlexBaranowski — COMMENTED** [review]
Hi @istiak101 

Thank you for Your PR ❤️. I would personally change repo urls/ remove modules comment and add change one repo name. I know that this looks like there is much, but all changes are easy to implement.

Once more, thank You - You made my day 😸 .

Best,
Alex

File: mock-core-configs/etc/mock/templates/eurolinux-9.tpl [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+[appstream-all]
+name=EuroLinux 9 AppStream All
+baseurl=https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/$basearch/certify-AppStream/all/
+gpgcheck=1
+enabled=1
+gpgkey=file:///usr/share/distribution-gpg-keys/eurolinux/RPM-GPG-KEY-eurolinux9
+
+[powertools-all]
+name=EuroLinux 9 PowerTools All
+baseurl=https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/$basearch/certify-PowerTools/all/
```
[the remark this answers is not in the document]
Comment (istiak101) [reply mock-core-configs/etc/mock/templates/eurolinux-9.tpl]: I almost forgot that in EnterPrise Linux 9 all distro changed name from PowerTools to CRB. Renamed the repository and corrected the URL.

**Comment by istiak101** [comment]
Hello @AlexBaranowski,

Thanks for your review. The `certify` prefix is removed from all repository. And renamed PowerTools to CodeReady Linux Builder.

And thanks for the coffee. Much appreciated. ❤️

**Review by xsuchy — APPROVED** [review]

**Comment by xsuchy** [comment]
LGTM. Unless someone objects, I will merge this on Monday.

**Comment by praiskup** [comment]
Running tests in EC2, us-east-1, and getting:

```
EuroLinux 9 BaseOS All                          109  B/s | 153  B     00:01
Errors during downloading metadata for repository 'baseos-all':
  - Status code: 404 for https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/i386/BaseOS/all/repodata/repomd.xml (IP: 185.93.1.244)
Error: Failed to download metadata for repo 'baseos-all': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

Only for the i686 repos, x86_64 is fine.

**Comment by xsuchy** [comment]
The correct URL seems to be https://fbi.cdn.euro-linux.com/dist/eurolinux/server/9/i686/BaseOS/all/repodata/repomd.xml
ie. s/i386/i686/

**Comment by praiskup** [comment]
Indeed, good catch.  Though I would swear I tried the very same config locally yesterday and it worked.  Hmm, going to fix this.

**Comment by praiskup** [comment]
@istiak101 @AlexBaranowski could the CDN be fixed similarly to the Euro Linux 8?  Create the i686 symlink to i386?

**Comment by istiak101** [comment]
> @istiak101 @AlexBaranowski could the CDN be fixed similarly to the Euro Linux 8? Create the i686 symlink to i386?

I don't have necessary permission to do this. Maybe @AlexBaranowski or his team members can create a symlink for this.

**Comment by AlexBaranowski** [comment]
Hi, I will fix it tommorrow, then Ping this issue.

Best,
Alex

**Comment by AlexBaranowski** [comment]
@istiak101 @istiak101 @praiskup symlink fixed

**Comment by praiskup** [comment]
Thank you, symlink works!  Remaining issue for i386:

```
EuroLinux 9 BaseOS All                          7.2 MB/s | 5.9 MB     00:00
EuroLinux 9 AppStream All                       2.3 MB/s | 1.5 MB     00:00
EuroLinux 9 CodeReady Linux Builder All         1.1 MB/s | 659 kB     00:00
No match for argument: patch
Error: Unable to find a match: patch
```

**Comment by AlexBaranowski** [comment]
I will fix it tommorow:

```
[repo@repo ~]# find /build/repo/eurolinux/9/ -name 'patch*.i686*'
/build/repo/eurolinux/9/beta/AppStream/i686/all/Packages/p/patch-2.7.6-16.el9.i686.rpm
/build/repo/eurolinux/9/beta/AppStream/i686/all/Packages/p/patchutils-0.4.2-7.el9.i686.rpm
```
We use different system than Koji, so composing ATM is not as good as it should be.

**Pull request merged**